src/Cake.Core/IO/ILaunchInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Cake.Core.IO
+{
+    /// <summary>
+    /// Represents a launch info.
+    /// </summary>
+    public interface ILaunchInfo
```

Comment:
Let's move this file to src/Cake.Core to be side-by-side with CakeEnvironment which is the class that uses it